pallets/avn-transaction-payment/src/tests/adjustment_fee_tests.rs

```diff
@@ -0,0 +1,582 @@
+use super::*;
+use crate::mock::{
+    event_emitted, new_test_ext, roll_one_block, AccountId, Balances, RuntimeCall, RuntimeEvent,
+    RuntimeOrigin, System, TestAccount, TestRuntime,
+};
+
+use frame_support::{dispatch::DispatchInfo, pallet_prelude::Weight};
+use pallet_transaction_payment::ChargeTransactionPayment;
+use sp_runtime::traits::SignedExtension;
+
+use frame_support::assert_ok;
+
+pub const TX_LEN: usize = 1;
+pub const INITIAL_SENDER_BALANCE: u128 = 200;
+pub const BASE_FEE: u32 = 14;
+pub const FIXED_FEE: u128 = 10;
+pub const PERCENTAGE_FEE: u32 = 50;
```

Comment:
Let's use something different please. 30 for example. What I want to see is: the discount rate is 30%, the fee we paid is 70% of the original. 50% does not allow us to disambiguate this.